web/components/SpeakerProfile.js

```diff
@@ -90,7 +90,15 @@ export const SpeakerProfile = ({
           )}
           {user && user.website && (
             <li>
-              <a href={user.website.startsWith('http') ? user.website : `https://${user.website}`}>Website</a>
+              <a
```

Comment:
This is from running "npm run fix-lint", included it in this PR just because it was easy as it had to be fixed